lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.image} />
```

Comment:
When using an `img` tag, it's good practice to always add na `alt=""` in case the image is broken. It's also more semantic.